tests/unit/views/GeneralReception.spec.js

```diff
@@ -100,6 +101,11 @@ describe('GeneralReception', () => {
       expect(wrapper.find('[data-type=pipeline-list]').findAll('option').length).toBe(1)
       expect(wrapper.find('[data-type=pipeline-list]').findAll('option')[0].text()).toBe('ONT')
     })
+    it('only shows Pacbio for Sequencescape Kinnex Tubes', async () => {
+      await wrapper.find('[data-type=source-list]').findAll('option')[3].setSelected()
```

Comment:
Probably better to find the option from the name rather than the index as the indexes could change, but I can see its done that way for the other tests.

Reply:
Good suggestion. Done 👍